### DISCUSSION TITLE
good afternoon can you help with plelistom smart (creating it)

### DISCUSSION THREAD

**Discussion by mrSaT13** [navidrome/navidrome#3092] [General]
I wrote a script for a smart playlist
even a few
but when adding it to the music folder
they don't appear can you help.maybe I didn't specify something correctly
1:
{
  "name": "TopPlayedTracks",
  "comment": "Tracks most frequently played",
  "maxSize": 200,
  "rules": [
    {
      "type": "playCount",
      "operator": "greaterThan",
      "value": "0"
    }
  ],
  "order": "playCount",
  "orderByDescending": true
}
2:
{
  "Name": "DailyPlaylist",
  "Description": "A daily updated playlist based on liked artists and songs",
  "Rules": [
    {
      "Rule": "OR",
      "Conditions": [
        {
          "Field": "Liked",
          "Operation": "EQUALS",
          "Value": "true"
        },
        {
          "Field": "ArtistLiked",
          "Operation": "EQUALS",
          "Value": "true"
        }
      ]
    }
  ],
  "UpdateInterval": "daily",
  "Limit": 25
}

3:
  "name": "Deja Vu",
  "comment": "Tracks that have never been played",
  "all": [
    {"eq": {"playCount": 0}}
  ],
  "sort": "artist",
  "order": "asc",
  "limit": 300
}

**Comment by deluan** [comment]
The syntax in these playlists is wrong. Check some examples and valid operators/fields here: https://github.com/navidrome/navidrome/issues/1417

Also, check the logs when importing the .NSP files, they usually says what is wrong with the file.

And lastly, there is a visual editor in the [Feishin](https://feishin.vercel.app/) client. Just point it to your Navidrome instance, create your smart playlists there, and they will appear and work in Navidrome as well.

**Comment by mrSaT13** [comment]
thank you very much, I will definitely try)
